Update docs to latest

waynesun09 opened the pull request:
$ sphinx-build doc/ docs/

Signed-off-by: Wayne Sun <gsun@redhat.com>

leelavg commented:
- quick question, there's `gen_docs.sh` script which regenerates everything and removes creds from docs, as I see less diff I assume current command only updates outdated content (which is good btw)?

waynesun09 committed:
Fix url not found error

The url is not generated in the xml now so check whether it could be
found before replace.

Also update the script with shellcheck fix.

Signed-off-by: Wayne Sun <gsun@redhat.com>

waynesun09 committed:
Fix sphinx-build error on indentation

Signed-off-by: Wayne Sun <gsun@redhat.com>

waynesun09 committed:
Rebuilt doc to latest

Signed-off-by: Wayne Sun <gsun@redhat.com>

waynesun09 commented:
@leelavg Fixed the issues with `gen_docs.sh` and updated the docs with the script

leelavg reviewed (approved):
- reviewed only gen_docs.sh and it lgtm
- either we need to find another way to generate docs or remove connection dependency to generate docs (mostly not possible i guess)
